pyccl/tests/test_hmfunc.py

```diff
@@ -153,3 +156,35 @@ def test_mass_function_mass_def_strict_always_raises():
     mdef = ccl.halos.MassDef(400, "critical")
     with pytest.raises(ValueError):
         ccl.halos.MassFuncBocquet16(mass_def=mdef, mass_def_strict=False)
+
+
+def test_nM_bocquet20_compare():
+    # Check that the values are sensible (they don't depart from other
+    # parametrisations by more than ~10%
+    Ms = np.geomspace(1E14, 1E15, 128)
+    mf1 = MF_emu
+    mf2 = ccl.halos.MassFuncTinker08(mass_def='200c')
+
+    nM1 = mf1(COSMO, Ms, 1.0)
+    nM2 = mf2(COSMO, Ms, 1.0)
+    assert np.allclose(nM1, nM2, atol=0, rtol=0.1)
+
+
+def test_nM_bocquet20_raises():
+    Ms = np.geomspace(1E12, 1E17, 128)
+
+    # Need A_s
```

Comment:
If I understand it correctly, you need sigma8 for the emulator. So the comment would be "need sigma8"?

Reply:
Yep, you're right. Done!